waku/waku_test.go

```diff
@@ -430,12 +427,6 @@ func TestSymKeyManagement(t *testing.T) {
 	if w.HasSymKey(id2) {
 		t.Fatalf("failed to delete second key: still exist.")
 	}
-	if k1 != nil {
-		t.Fatalf("failed to delete second key: first key is not nil.")
-	}
-	if k2 != nil {
-		t.Fatalf("failed to delete second key: second key is not nil.")
-	}
```

Comment:
redundant `err` check. Both are checked on L-421 and are not updated before this line.